CODE_OF_CONDUCT.md

```diff
@@ -40,7 +40,7 @@ Talking about them from a outside perspective is allowed, for example discussing
 
 ### Safe Space
 
-This community is explicitly a safe space for the following people:
+This community is explicitly a safe space for marginalised and under-represented groups of people such as:
```

Comment:
@lexika979 Laund & I had a chat on a side channel and we figured out modified language, what do you think of this?